CMakeLists.txt

```diff
@@ -38,7 +38,7 @@ execute_process(
   OUTPUT_VARIABLE LLVM_SYSTEM_LIBS
   OUTPUT_STRIP_TRAILING_WHITESPACE
 )
-set(LLVM_LIBS "${LLVM_LIBS} ${LLVM_SYSTEM_LIBS}")
+set(LLVM_LIBS "${LLVM_LIBS} ${LLVM_SYSTEM_LIBS} ${CMAKE_SOURCE_DIR}/third_party/llvm/${CMAKE_BUILD_TYPE}/lib/libclangASTMatchers.a")
```

Comment:
this isn't the right solution, see how we set LLVM_LIBS in the first place and see if there's a flag that can be added there which gets the correct libraries

Reply:
llvm-config does not seem to have a relevant flag

Reply:
well, I assert that there is a solution to this which is different from bypassing llvm-config completely